frameworks/elastic/universe/marathon.json.mustache

```diff
@@ -96,7 +95,6 @@
     {{#service.region}}
     "SERVICE_REGION": "{{service.region}}",
     {{/service.region}}
-    "TASKCFG_ALL_XPACK_ENABLED": "{{elasticsearch.xpack_enabled}}",
```

Comment:
I do not see a change that removes (or at least updates the description) of the `elasticsearch.xpack_enabled` option in `config.json`.

Reply:
Remove x-pack enable condition from config.json file as it is inbuild in es 6.3.1 
Note: some security feature are not supportable/enable on basic & trial license.

Reply:
> Note: some security feature are not supportable/enable on basic & trial license.

Yes, I was wondering about this. How are we allowing users to toggle this on and off?